Yoast/wordpress-seo
# Reorganize search appearance

⚠️ NOT FINISHED, STYLING + COLLAPSING NEEDS TO BE BUILT ⚠️ 
## Summary

This PR can be summarized in the following changelog entry:

* Show post type settings and their archive settings below each other, instead of first showing all post types and then all archives.

## Relevant technical choices:

* No setting names have been changed, I've only moved code around to bring these things together.

## Test instructions

Please test very well to make sure all settings are properly stored and also still are accessible when they're set on a lower version and then switched to this version.

## Quality assurance


Fixes #10063
